emild_config:
    TCP_IP: '192.168.1.177'
    TCP_PORT: 9001
    BUFFER_SIZE: 1024
    timeout_counter: 0
    timeout_threshold: 5.0
    frame_id: 'map' 